src/app/components/GameRun.tsx

```diff
@@ -1,31 +1,40 @@
-import React, { ReactNode, useState } from 'react'
-import { Game } from '../page'
-import Timer from './Timer'
-import TeamScorer from './TeamScorer'
-
-
+import React, { ReactNode, useState } from "react";
+import { Game } from "../page";
+import Timer from "./Timer";
+import TeamScorer from "./TeamScorer";
 
 type Props = {
-    gameSetup: Game;
-    setCurrentInterval: React.Dispatch<React.SetStateAction<number>>;
-    currentInterval: Number
```

Comment:
All primitive data types are lower case, this is the Number wrapper type which is kinda hard to explain but it is used for type conversion and stuff like that.